test/sandbox_test.py

```diff
@@ -105,3 +105,49 @@ def test_sandbox_terminate(client, servicer):
         sb.terminate()
 
         assert sb.returncode != 0
+
+
+@skip_non_linux
+@pytest.mark.asyncio
+async def test_sandbox_stdin_async(client, servicer):
```

Comment:
I have an `async` test and a `sync` test that pretty much does the same thing. Should I remove one of them?